google-cloud-trace/docs/toc.json

```diff
@@ -1,8 +1,71 @@
 {
-  "guides": [],
+  "guides": [
+    {
+      "title": "Troubleshooting",
+      "id": "troubleshooting",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/troubleshooting/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/troubleshooting/readme.md"
+    },
+    {
+      "title": "Contributing",
+      "id": "contributing",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/contributing/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/contributing/readme.md"
+    }
+  ],
   "services": [
     {
-      "title": "TraceServiceApi",
+      "title": "Google::Cloud",
+      "type": "google/cloud"
+    },
+    {
+      "title": "Google::Cloud::Trace",
+      "type": "google/cloud/trace",
+      "nav": [
```

Comment:
@dazuma Our tactic has been to make sure that "secondary" classes are accessible (linked) as return types or params. But the site is lacking both index and search, so I fully understand if you feel it is the best decision to just list them all in the nav.